spec/functional_tests/documents/my_documents/my_documents_actions_spec.rb

```diff
@@ -10,6 +10,9 @@
 new_spreadsheet = "New_Spreadsheet_#{SecureRandom.hex(5)}"
 new_presentation = "New_Presentation_#{SecureRandom.hex(5)}"
 new_folder = "New_Folder_#{SecureRandom.hex(5)}"
+new_form_blank = "New_Form_Blank_#{SecureRandom.hex(5)}"
```

Comment:
Since there is a lot of construction like those - maybe it's better to create helper method

```
def generate_random_name(pattern)
  "#{pattern}_#{SecureRandom.hex(5)"
end

new_form_blank = generate_random_name('New_Form_Blank')

Reply:
For other specs I will add generating documents names in separate PR